### DISCUSSION TITLE
Need Last Checkpoint

### DISCUSSION THREAD

**Discussion by ankan8145** [Vision-CAIR/MiniGPT-4#332] [Q&A]
Your MiniGPT-4 is just Awesome and I am very much interested in using this and fine-tuning for my work. But I have one problem, I do not have enough resources to train the model, Can you please provide the last checkpoint which is used for hosting the app "https://huggingface.co/spaces/Vision-CAIR/minigpt4" here? Or from "minigpt4/train_configs/minigpt4_stage2_align.yaml" I see that you are using "/ibex/project/c2133/vicuna_jun_checkpoint_wihtout_prompt/20230412162/checkpoint_3.pth" checkpoint to train the model for the second stage, it will be helpful if you provide that also.
Here is my Email_id - ankanderia01@gmail.com

**Comment by TsuTikgiau** [comment]
Hello! the checkpoint used to host the app is the 7B vicuna version in the Readme and you can download it from the readme here

<img width="1085" alt="image" src="https://github.com/Vision-CAIR/MiniGPT-4/assets/13706888/ba1d0f38-4017-4705-a363-add25e82c033">

The ckpt of the first stage is also provided in the readme, shown here

<img width="1057" alt="image" src="https://github.com/Vision-CAIR/MiniGPT-4/assets/13706888/72d8e0d7-c312-49d7-8b48-0c1b765e1ec9">

Thank you!

**Reply by ankan8145** [reply]
Thanks for your reply. Can you provide the checkpoint of the 2nd stage ?

**Reply by TsuTikgiau** [reply]
Hello, the ckpt after the second stage is the final ckpt we released in the readme here

![image](https://github.com/Vision-CAIR/MiniGPT-4/assets/13706888/87d97f62-2d88-40e6-815c-450f63566fd7)
